fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -5226,6 +5226,10 @@ admin_stmt ::=
     {:
         RESULT = new AdminCancelRepairTableStmt(table_ref);
     :}
+    | KW_ADMIN KW_COMPACT KW_TABLE base_table_ref:table_ref opt_wild_where
```

Comment:
Add `KW_COMPACT` to the `keyword ::=` region

Reply:
> Add `KW_COMPACT` to the `keyword ::=` region

Done.